internal/server/iigointernal/orchestration.go

```diff
@@ -70,10 +72,26 @@ func RunIIGO(g *gamestate.GameState, clientMap *map[shared.ClientID]baseclient.C
 	}
 	iigoClients = *clientMap
 
+	// Increments the budget by a constant 100
+	// TODO:- the constant should be retrieved from the rules
+	g.IIGORolesBudget[shared.President] += 100
+	g.IIGORolesBudget[shared.Judge] += 100
+	g.IIGORolesBudget[shared.Speaker] += 100
+
+	// Pass in gamestate -
+	// So that we don't have to pass gamestate as arguments in every function in roles
+	judicialBranch.gameState = g
```

Comment:
Could we maybe have like a judicialBranch.syncWithGame(gamestate, iigoConfig) rather than direct assignment

Reply:
Done

Reply:
Cool thanks